polaris-react/src/components/Icon/Icon.tsx

```diff
@@ -6,37 +6,28 @@ import type {IconSource} from '../../types';
 
 import styles from './Icon.scss';
 
-type Color =
+type Tone =
   | 'base'
   | 'subdued'
+  | 'caution'
+  | 'warning'
   | 'critical'
   | 'interactive'
-  | 'warning'
   | 'highlight'
```

Comment:
```suggestion
  | 'info'
```

should highlight be info? It's the only one that doesn't match 1:1 with a color role

Reply:
Yeah! Good call